ci/update-checker/src/main/scala/com/twitter/intellij/updatechecker/UpdateChecker.scala

```diff
@@ -0,0 +1,93 @@
+package com.twitter.intellij.updatechecker
+
+import java.nio.file.Path
+
+import org.jetbrains.intellij.pluginRepository._
+import org.jetbrains.intellij.pluginRepository.model.{PluginUpdateBean, ProductFamily}
+import org.virtuslab.ideprobe.Config
+import org.virtuslab.ideprobe.Extensions.PathExtension
+import org.virtuslab.ideprobe.dependencies.Plugin
+import org.virtuslab.ideprobe.dependencies.Plugin.Versioned
+
+import scala.jdk.CollectionConverters._
+
+object UpdateChecker {
+
+  // Assuming this program is launched in the project's base directory
+  private val configFile = Path.of("tests/src/test/resources/versions.conf")
+
+  case class PluginUpdate(id: String, version: String)
+
+  private val updateBeanOrdering = Ordering.fromLessThan[PluginUpdateBean] {
+    case (x, y) => x.getCdate < y.getCdate
+  }
+
+  private def updatedConfig(oldConfig: String, changes: Seq[PluginUpdate]): String = {
+    changes.foldLeft(oldConfig){ case (config, PluginUpdate(id, version)) =>
+      config.replaceFirst(
+        s"""\\{ id = "$id", version = "[a-zA-Z0-9_\\-.]+"""",
+        s"""{ id = "$id", version = "$version""""
+      )
+    }
+  }
+
+  def main(args: Array[String]): Unit = {
```

Comment:
We should also update intellij itself

Reply:
I'll do that in one of the following pull requests.